cli/dcoscli/cluster/main.py

```diff
@@ -55,15 +55,21 @@ def _main(argv):
 
     http.silence_requests_warnings()
 
-    return cmds.execute(_cmds(), args)
+    return cmds.execute(_cmds(doc_name), args)
 
 
-def _cmds():
+def _cmds(doc_name):
     """
+    :param doc_name: the docopt help file in use
+    :type doc_name: str
     :returns: all the supported commands
     :rtype: list of dcos.cmds.Command
     """
 
+    list_arg_keys = ['--json', '--attached']
+    if doc_name == 'cluster_ee':
+        list_arg_keys.append('--linked')
```

Comment:
So here we'd have an option which is only for EE, this is a hack to have it not failing on Open. Should I add the `--linked` flag there too instead?

Reply:
This is fine for now.